app/index.jsx

```diff
@@ -61,25 +64,49 @@ const renderControls = () => (
         <ChartSetting />
     </React.Fragment>
 );
-
 const requestAPI = connectionManager.send.bind(connectionManager);
 const requestSubscribe = streamManager.subscribe.bind(streamManager);
 const requestForget = streamManager.forget.bind(streamManager);
 const shareOrigin = window.location.href.split('?')[0];
+function Notifier() {
```

Comment:
make this a class. Use a separate file, or make it part of Notification component